source/_integrations/diagnostics.markdown

```diff
@@ -11,14 +11,14 @@ ha_codeowners:
 ha_integration_type: system
 ---
 
-The diagnostics integration provides a way to download diagnostic data from
-a device or integration, that can be shared in issue reports. Sharing
+The diagnostics {% term integration %} provides a way to download diagnostic data from
+a {% term device %} or {% term integration %}, that can be shared in issue reports. Sharing
 diagnostics data when reporting an issue allows developers to diagnose 
 and fix your reported problem quicker.
 
 <p class='img'>
 <img class="no-shadow" src='/images/blog/2022-02/diagnostics.png' alt='Screenshot showing the Download Diagnostics button on a Sonos device page'>
-Screenshot of the Download Diagnostics button on the device page
+Screenshot of the Download diagnostics button on the device page
```

Comment:
```suggestion
Screenshot of the download diagnostics button on the device page
```
It's all uppercase in the dashboard, but we can use lowercase here to describe the image.